build.sbt

```diff
@@ -6,7 +6,7 @@ ThisBuild / organization := "org.typelevel"
 ThisBuild / organizationName := "Typelevel"
 ThisBuild / tlBaseVersion := "1.0"
 ThisBuild / scalaVersion := Scala213
-ThisBuild / crossScalaVersions := Seq(Scala212, Scala213, Scala3)
+ThisBuild / crossScalaVersions := Seq(Scala212, Scala3, Scala213)
```

Comment:
whooah, ordering changes the behavior? Looks like pretty thin ice.

Reply:
Yeah, you are right 😕

The problem is how to set the default scala version. If we use the `scalaVersion` variable the problem is it is changing all the time with `++` so the result is not stable. So for now it uses the last one in `crossScalaVersions`.

Maybe we need another setting, `defaultScalaVersion` or something. Do you have any ideas?

Reply:
If it needs only for the CI, maybe it's fine to have something specific like `tlciScalaVersion`? And the default value could be the last of `crossScalaVersions`.

Reply:
Thanks for pointing this out; I would definitely like to fix this.

Reply:
I opine that basically, it's good to have it in `sbt-typelevel`. It's more transparent behavior for end-users. But probably it's not a topic of this PR? I would like to have it merged as soon as possible :)

Reply:
Yes, for sure, this PR is ready 👍